app/models/customer.rb

```diff
@@ -0,0 +1,3 @@
+class Customer < ApplicationRecord 
+  validates_presence_of :name 
```

Comment:
This is where we can set up the many to many relationship. In this model, we can add 
```  
has_many :customers_items
 has_many :items, through: :customers_items
```

We want to do the same in the Item model, but for having many customers (i.e. the opposite)